### PR TITLE
adding docs for getting started page

### PR DESCRIPTION
**Pull request by ipberry**
## Description
Adding docs for getting started page

## Todos
Notable points that this PR has either accomplished or will accomplish.
  - [x] added docs for getting started page
  - [x] changed our default text color to dark blue, we had discussed this earlier, let me know your thoughts
  - [x] updated return type for run_sparcle in driver
  - [x] added user yaml to the base

### CODE REVIEW

File: docs/getting_started.rst [hunk 1]
```diff
@@ -1,9 +1,75 @@
 Getting Started
 ===============
+Installing sparcle_qc
+---------------------
+To get started, you will need to install sparcle_qc and its dependencies. We have provided a yaml file to create a conda environment containing all of the necessary packages.
 
+First, clone the `sparcle_qc repository <https://github.com/carolinesargent/sparcle_qc>`_
```
Comment (carolinesargent) [inline docs/getting_started.rst:7]: would be cool if we could make this code to be copied (git clone https:/github......)

File: docs/getting_started.rst [hunk 2]
```diff
@@ -1,9 +1,75 @@
 Getting Started
 ===============
+Installing sparcle_qc
+---------------------
+To get started, you will need to install sparcle_qc and its dependencies. We have provided a yaml file to create a conda environment containing all of the necessary packages.
```
Comment (carolinesargent) [inline docs/getting_started.rst:5]: Do we want to use sparcle_qc or Sparcle_QC when talking about it outside of code?

Comment (ipberry) [reply docs/getting_started.rst:5]: great point, I don't particularly mind either way, but maybe since our logo is Sparcle_QC, we should stick with that?

Comment (carolinesargent) [reply docs/getting_started.rst:5]: yes, I like that!

Comment (ipberry) [reply docs/getting_started.rst:5]: Updated

**Pull request merged**